h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -30,6 +32,11 @@
 
 @Ignore("Support for tests, but no actual tests here")
 public class TestUtil extends Iced {
+  {
```

Comment:
This is a test. Test should fail on assertions, so in the tests assertion flag should be always turned on. TestUtil ensures that the right level is there. We may think about setting the flag, but well, in this specific case the flag should be set.